roles/logins/tasks/main.yml

```diff
@@ -47,36 +47,44 @@
     mode: '0755'
   become: true
 
-- name: Install login_checks.sh script.
-  copy:
-    src: login_checks.sh
-    dest: /etc/pam-script.d/login_checks.sh
-    owner: root
-    group: root
-    mode: '0755'
-  when: inventory_hostname in groups['cluster']
-  become: true
-
-- name: Enable login_checks.sh script for ses_open.
+#
+# Login checks currently disabled,
+# because error handling/reporting no longer works on CentOS >= 7.x,
+# due to changes in the PAM stack.
+# Login checks were only used to create Slurm accounts in the Slurm accounting DB.
+# This functionality has been relocated to the Slurm job_submit.lua plugin,
+# which will now automatically create account, users and associations of slurm users to slurm accounts
+# upon job submission when they do not already exist.
+#
+#- name: Install login_checks.sh script.
+#  copy:
+#    src: login_checks.sh
+#    dest: /etc/pam-script.d/login_checks.sh
```

Comment:
Why leave so much in comment? It is in version control

Reply:
Finding it back in version control can be quite hard. Basically all the pam_script stuff is no longer used as it was only used for the ```login_checks.sh``` script, but it is nice to have pam_script configured and example code on how to use it, so it's easy to add additional scripts later on...